src/control-plane/backend/layer/lambda-web-adapter/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG TARGET_PLATFORM=linux/amd64
+FROM --platform=$TARGET_PLATFORM public.ecr.aws/amazonlinux/amazonlinux:2 as build-stage
+ARG ARCH=x86_64
+RUN rpm --rebuilddb && yum install -y yum-plugin-ovl jq
+RUN yum groupinstall -y "Development tools"
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN source $HOME/.cargo/env && rustup target add ${ARCH}-unknown-linux-musl
+RUN curl -k -o /${ARCH}-linux-musl-cross.tgz https://musl.cc/${ARCH}-linux-musl-cross.tgz \
+        && tar zxf /${ARCH}-linux-musl-cross.tgz \
+        && ln -s /${ARCH}-linux-musl-cross/bin/${ARCH}-linux-musl-gcc /usr/local/bin/${ARCH}-unknown-linux-musl-gcc
+WORKDIR /app
+RUN git clone https://github.com/awslabs/aws-lambda-web-adapter.git &&\
```

Comment:
Let's pin to a certained version(latest is fine for now). You can define an `ARG` as version.

Reply:
Add `ARG` to **Dockerfile**, and git checkout to version tag. If version is `latest`, will checkout the latest tag.